mlf_core/create/template_creator.py

```diff
@@ -50,7 +51,10 @@ def process_common_operations(self, path: Path, skip_common_files=False, skip_fi
         """
         # create the common files and copy them into the templates directory (skip if flag is set)
         if not skip_common_files:
```

Comment:
wondering if we even need `skip_common_files` here since we dont have templates, that dont require the common files like in cookietemple

Reply:
I don't think that we need it, yes